integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDiagnosticsFailedCondition.java

```diff
@@ -0,0 +1,668 @@
+  void testReplicasTooHigh() {
+    String image = MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG;
+
+    logger.info("Creating domain resource with replicas=100");
+    Domain domain = createDomainResource(domainUid, domainNamespace, adminSecretName,
+        OCIR_SECRET_NAME, encryptionSecretName, 100, image);
+
+    try {
+      logger.info("Creating domain");
+      createDomainAndVerify(domain, domainNamespace);
+
+      //check the desired completed, available and failed statuses
+      checkStatus("True", "False", "True");
+
+    } finally {
+      deleteDomainResource(domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Test domain status condition with non-existing image. Verify the following conditions are
+   * generated: type: Completed, status: true type: Available, status: false type: Failed status: true
+   */
+  @Test
+  @DisplayName("Test domain status failed condition with non-existing image")
+  void testImageDoesnotExist() {
+    String image = MII_BASIC_IMAGE_NAME + ":non-existing";
+
+    logger.info("Creating domain resource with non-existing image");
+    Domain domain = createDomainResource(domainUid, domainNamespace, adminSecretName,
+        OCIR_SECRET_NAME, encryptionSecretName, replicaCount, image);
+
+    try {
+      logger.info("Creating domain");
+      createDomainAndVerify(domain, domainNamespace);
+
+      //check the desired completed, available and failed statuses
+      checkStatus("True", "False", "True");
+
+    } finally {
+      deleteDomainResource(domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Test domain status failed condition with missing image pull secret. Verify the following conditions are
+   * generated: type: Completed, status: true type: Available, status: false type: Failed status: true
+   */
+  @Test
+  @DisplayName("Test domain status condition with missing image pull secret")
+  void testImagePullSecretDoesnotExist() {
+    String image = MII_BASIC_IMAGE_NAME + ":" + MII_BASIC_IMAGE_TAG;
+
+    logger.info("Creating domain resource with missing image pull secret");
+    Domain domain = createDomainResource(domainUid, domainNamespace, adminSecretName,
+        OCIR_SECRET_NAME + "bad", encryptionSecretName, 100, image);
+
+    try {
+      logger.info("Creating domain");
+      createDomainAndVerify(domain, domainNamespace);
+
+      //check the desired completed, available and failed statuses
+      checkStatus("True", "False", "True");
+
+    } finally {
+      deleteDomainResource(domainUid, domainNamespace);
+    }
+  }
+
+  /**
+   * Test domain status failed condition with incorrect image pull secret. Verify the following conditions are
+   * generated: type: Completed, status: true type: Available, status: false type: Failed status: true
+   */
+  @Test
+  @DisplayName("Test domain status condition with incorrect image pull secret")
+  void testIncorrectImagePullSecret() {
```

Comment:
Here we are generating two Error conditions (a) bad image name (b) bad image pull secret.  We should create one error condition per usecase. May be we should have different test case that  generate multiple error conditions

Reply:
that was a mistake, using the correct image now